src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal static class ConfigPathsRunner
+    {
+        public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
+        {
+            var settings = RunnerHelper.GetSettingsFromDirectory(args.WorkingDirectory);
+            ILogger logger = getLogger();
+
+            var filePaths = settings.GetConfigFilePaths();
+            foreach (var filePath in filePaths)
+            {
+                logger.LogMinimal(filePath);
+            }
+        }
+    }
+
+    internal static class RunnerHelper
+    {
+        public static ISettings GetSettingsFromDirectory(string directory)
+        {
+            if (string.IsNullOrEmpty(directory))
```

Comment:
I think a more natural implementation might be to do this check outside of the utility method for getting the settings. 

The name is `GetSettingsFromDirectory`, but it accepts null, which is not obvious from the method signature.